holidayextras/ui-brand-chessington
# changing row calculation to pixels to avoid calculator error pushing …

#### What does this PR do? (please provide any background)
There was an error in the calculation of the chessington row. This was spotted in the pattern library, seems the theme was built in a previous version of bootstrap which led to a REM vs PX calculation breaking the grid.

#### What tests does this PR have?
None

#### How can this be tested?
npm link or install this branch inside the pattern library. Check out ```<div class="row">``` it should not have a negative margin.

#### What gif best describes how you feel about this work?
![giphy](https://cloud.githubusercontent.com/assets/6357978/12169856/d322f396-b533-11e5-967a-d2ecb120941a.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
